static/js/components/SiteSidebar.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from "react"
+import { NavLink } from "react-router-dom"
+
+import { siteComponentUrl, siteUrl } from "../lib/urls"
+
+import { ConfigItem, Website } from "../types/websites"
+
+interface Props {
+  website: Website
+}
+export default function SiteSidebar(props: Props): JSX.Element {
+  const { website } = props
+
+  const configItems: ConfigItem[] = website.starter?.config?.collections ?? []
+
+  return (
+    <div className="sidebar">
+      <ul>
+        <li>
+          <NavLink exact to={siteUrl(website.name)} activeClassName={"active"}>
+            Content
+          </NavLink>
```

Comment:
Sure, but which wireframes? The one linked in #40 still shows "Content"

Reply:
@noisecapella Ah, these must have changed since I looked at them last. I still think it's a good idea to get rid of this since we'll probably have child menu items for each content type